ast/builtins.go

```diff
@@ -2327,6 +2328,51 @@ with all others being treated as intermediates.`,
 	),
 }
 
+var CryptoX509ParseAndVerifyCertificatesWithOptions = &Builtin{
+	Name: "crypto.x509.parse_and_verify_certificates_with_options",
+	Description: `Returns one or more certificates from the given string containing PEM
+or base64 encoded DER certificates after verifying the supplied certificates form a complete
+certificate chain back to a trusted root. Second argument is a option object to pass
+extra configs to verify the validity of certificates.
+
+The first certificate is treated as the root and the last is treated as the leaf,
+with all others being treated as intermediates.` +
```

Comment:
```suggestion
with all others being treated as intermediates.

` +
```
<img width="232" alt="Screenshot 2024-02-21 at 18 11 56" src="https://github.com/open-policy-agent/opa/assets/1774239/829ce6e9-5af9-412a-8353-d7a5500322e4">